go.mod

```diff
@@ -10,14 +10,14 @@ require (
 	github.com/json-iterator/go v1.1.6 // indirect
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.1 // indirect
-	github.com/pulumi/pulumi v1.10.1
-	github.com/stretchr/testify v1.4.1-0.20191106224347-f1bd0923b832
+	github.com/pulumi/pulumi/pkg v0.0.0-20200328214918-6a945294e309
```

Comment:
since we did an official release are you able to update these?

```
github.com/pulumi/pulumi/pkg v1.13.1
github.com/pulumi/pulumi/sdk v1.13.1
```

Reply:
Done.

> did any of the code you're importing from pulumi/pulumi move?

only had to update `apitype` in the CI test utils

![image](https://user-images.githubusercontent.com/1112768/77953155-ac94c500-7281-11ea-9d9e-cc4eeb606f91.png)